cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from .models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=100)
+    description = serializers.CharField(max_length=1000)
```

Comment:
The `description` field in the serializer has a `max_length` of 1000, but the model defines it as a `TextField`, which does not have a `max_length` constraint. Consider removing the `max_length` from the serializer to align with the model's field definition.